tendrl/commons/flows/create_cluster/ceph_help.py

```diff
@@ -4,67 +4,53 @@
 
```

Comment:
remove this file and any other ceph file

Reply:
i will remove it